docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md

```diff
@@ -1 +1,97 @@
-# Shaka Packager
\ No newline at end of file
+# Shaka Packager
+
+## Setup using Docker
+
+- First download and install Docker - https://www.docker.com/get-started
+
+- Pull the latest Shaka Packager Docker image by entering ```docker pull google/shaka-packager``` in your terminal.
+
+- Now run the Docker image by entering ```docker run -v /<file_path_to_content>/:/media -it --rm google/shaka-packager``` in your terminal. This will replicate your directory and create a ```media``` file path within the Docker container.
+
+- Test that setup is correct, by analyising a file within the filepath. You can do this by entering the command ```packager input=/media/<name_of_content>.mp4 --dump_stream_info```.
+
+If successful you should then get a dump of the stream information which should look something like this:
+
+```
+File "/media/test.mp4":
+Found 1 stream(s).
+Stream [0] type: Video
+ codec_string: avc1.640028
+ time_scale: 12288
+ duration: 17078784 (1389.9 seconds)
+ is_encrypted: false
+ codec: H264
+ width: 1920
+ height: 810
+ pixel_aspect_ratio: 1:1
+ trick_play_factor: 0
+ nalu_length_size: 4
+
+Packaging completed successfully.
+```
+
+## Get CPIX Documents
+
+CPIX Documents are required to give Shaka Packager the necessary information in order to encrypt your media.
+
+### Using VUDRM Admin site
```

Comment:
what is this section for? You do not need multiple examples of how to _get_ a cpix doc in this doc as the purpose of this documentation is to show how you _use_ a cpix doc.

Reply:
Actioned.